src/mixins/legal-api-mixin.ts

```diff
@@ -75,6 +75,29 @@ export default class LegalApiMixin extends Vue {
       })
   }
 
+  /**
+   * Fetches a filing.
+   * @returns a promise to return the filing of the specified type, or null if not found
+   */
+  async fetchFiling (businessId: string, filingType: string): Promise<IncorporationFilingIF> {
```

Comment:
This function will most likely need to pass an ID or something in the future as well, if in the event that we are pulling on multiple filings of the same type. Ie.. pulling an Address Change for correction, will require more logic.